widgets/explore2.py

```
@@ -0,0 +1,19 @@
+from starlette.requests import Request
+
+from daras_ai_v2.gooey_builder import render_explore_builder_prompt
+from daras_ai_v2.meta_content import raw_build_meta_tags
+
+META_TITLE = "Explore | Gooey.AI"
+META_DESCRIPTION = "Find, fork and run your field’s favorite AI recipes on Gooey.AI"
```

Ruff (0.15.14)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)